index.html

```diff
@@ -1,31 +1,71 @@
 <html>
-  <script src="try-slang.js"> </script>
+<style>
+textarea {
```

Comment:
As a cleanup, we should put the css file in a separate location and reference it from the html.